.circleci/config.yml

```diff
@@ -57,7 +57,7 @@ jobs:
                 command: ./ci/run_travis_commands.sh
                 environment:
                     CMAKE_BUILD_TYPE: Debug
-                    FLAGS: -Wall -Wunused-parameter -Werror -Werror=sign-compare -Wunused-lambda-capture -fsanitize=signed-integer-overflow
+                    FLAGS: -Wall -Wextra -Wno-ignored-qualifiers -Werror -Werror=sign-compare -Wunused-lambda-capture -fsanitize=signed-integer-overflow
```

Comment:
@aehyvari , do you know why we have the extra flags like `unused-lambda-capture` here but not anywhere else?

Reply:
OK, we could try to unify the flags later. It would be good to also have a CI run with sanitizers, I think. But all this can be done in some later PR.